### PR TITLE
.netstandard2_0 support

### PR DESCRIPTION
**Pull request by chriswill**
I added support to multi-target .net standard 2.0 and .net 4.5.  It is possible to target down to .net standard 1.3 as well, but that requires an additional dependency on System.Diagnostics.Process from nuget.org and I'm not sure if that's necessary.

The change involved taking out the old csproj, and then adding back a new style csproj targeting the two frameworks.  This now requires dotnet to build, so build.cmd has been updated.  You can also just build from Visual Studio.

The existing test project was not changed and all tests run successfully.

### CODE REVIEW

**Comment by chriswill** [comment]
Looks like it builds successfully on AppVeyor :)

**Comment by kendaleiv** [comment]
Nice! Could we also update the `Core.Tests` project? See https://github.com/ritterim/silverpop-dotnet-api/blob/2d76d204f1b16222b7bd6da79c1ee5f97401e316/test/Silverpop.Core.Tests/Silverpop.Core.Tests.csproj for an example.

**Comment by chriswill** [comment]
I'll make an additional update later today.

**Comment by chriswill** [comment]
I updated the test project to support multi-targeting.  It got a lot more async than the original because the Microsoft Azure Storage Library is async-only for .netstandard2_0.  But all tests pass in both .Net 4.5 and .netstandard2_0.

File: build.cmd [hunk 1]
```diff
@@ -28,7 +18,7 @@ if not exist .nuget @md .nuget
 @copy %CACHED_NUGET% .nuget\nuget.exe > nul
 
 :Build
-%MSBUILD% build\Build.proj /m /v:m %* /fl /flp:LogFile=msbuild.log;Verbosity=Detailed /nr:false /nologo
+dotnet build src/core/core.csproj --configuration release
```
Comment (kendaleiv) [inline build.cmd]: Currently https://github.com/ritterim/automation-storage-emulator/blob/6340f0d5487f36ce809798ad55840b3898404152/build/Build.proj is responsible for building, testing, and packaging, with the latter two currently missing from this PR.

Want to add **running tests** and **creating the NuGet package** as part of `build.cmd`? If not, I can merge this into a branch and finish that up.

Excellent contribution overall. 🎉

**Comment by chriswill** [comment]
I updated the build script, using the netcore services for building, testing and packaging.  Along the way I had to update the app to reference .net 4.5.2 because xunit no longer supports testing .net 4.5 apps [link](https://github.com/xunit/xunit/issues/1138#issuecomment-284748337).

The previous build script had some errors dealing with paths that had a space in them, fyi.  This was resolved by enclosing them in quotes.

**Comment by kendaleiv** [comment]
Thanks for this awesome contribution! AppVeyor isn't picking up the artifact but I'll figure it out.

**Comment by chriswill** [comment]
I'm not familiar with AppVeyor's build process, so sorry can't be of help there.  If you have to put the output into a specific directory use the --output <OUTPUT_DIRECTORY> param on the command line for the dotnet pack command.

**Comment by chriswill** [comment]
Would appreciate if you can update Nuget, as I would like to pull in an update that references .net standard :)

**Comment by chriswill** [comment]
This is the command line that I think sets up your AppVeyor artifact support:

```
dotnet pack src/core/core.csproj --configuration release --output .\..\..\artifacts
```

**Comment by kendaleiv** [comment]
See #20 which should automatically deploy when merged.

**Pull request merged**